Makefile

```diff
@@ -1,8 +1,39 @@
 PROJ_VERSION = 6.3.0
-
-install_proj:
+install_proj: ## Install PROJ and clang (requirements to use proj crate)
 	sudo apt update
-	sudo apt install -y wget build-essential pkg-config sqlite3 libsqlite3-dev clang
-	wget https://github.com/OSGeo/proj.4/releases/download/$(PROJ_VERSION)/proj-$(PROJ_VERSION).tar.gz && tar -xzvf proj-$(PROJ_VERSION).tar.gz
-	cd proj-$(PROJ_VERSION) && ./configure --prefix=/usr && make && sudo make install
+	sudo apt install -y clang
+
+	# Needed only for proj install
+	sudo apt install -y wget build-essential pkg-config sqlite3 libsqlite3-dev
+
+	# remove PROJ system version from packages
+	sudo apt remove libproj-dev
+
+	wget https://github.com/OSGeo/proj.4/releases/download/$(PROJ_VERSION)/proj-$(PROJ_VERSION).tar.gz
+	tar -xzvf proj-$(PROJ_VERSION).tar.gz
+	pushd proj-$(PROJ_VERSION)
+	./configure --prefix=/usr && make
+	sudo make install
+	popd
 	rm -rf proj-$(PROJ_VERSION) proj-$(PROJ_VERSION).tar.gz
+
+fmt: format ## Check formatting of the code (alias for 'format')
+format: ## Check formatting of the code
+	cargo fmt --all -- --check
+
+clippy: lint ## Check quality of the code (alias for 'lint')
+lint: ## Check quality of the code
+	cargo clippy --workspace --all-features --all-targets -- --warn clippy::cargo --allow clippy::multiple_crate_versions --deny warnings
+
+test: ## Launch all tests
+	# Run all the tests of `transit_model` in the entire repository,
+	# activating all features (including `xmllint`), then without features
+	# to make sure that both work
+	cargo test --workspace --all-features
```

Comment:
Wouldn’t a check be useful before launching tests (or clippy) to verify that proj is well installed (and in 6.3) ? :thinking:

Reply:
It's a bit outside the initial scope of the PR, but why not.

I wouldn't do it on clippy as someone may want to work without the proj feature (so avoid netex).

As for the tests, it won't work without proj, so it might be a good hint to just check that.
But is it enough to check that cmd `proj` returns a version number?

I wouldn't pin a version, as there are probably multiple ones that are compatible (using 6.3.1 locally), and it's nicer to be able to test them and still use `make test`.

Reply:
As discussed: not adding a check as the ratio value/effort is not clear.